tests/kustomize_build.sh

```diff
@@ -79,3 +79,61 @@ kpt fn source examples |
   kpt fn sink .
 assert_contains_string configmap_the-map.yaml "app: hello"
 assert_dir_exists knative-serving
+
+testcase "kpt_kustomize_build_fn_plugins"
+mkdir fn
+mkdir -p source/base
+mkdir -p source/kustomize/plugin/
```

Comment:
Where do you use the plugin dir? Is it implicit when setting KUSTOMIZE_PLUGIN_HOME?

Reply:
yes, correct: [here](https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/106/files#diff-cddb7e3d19c09d9f69d411848ad17f9cR103) That's because kpt-function based plugins can only work when `--enable_alpha_plugins` is set in kustomize options. And if this flag is set it's necessary to have a directory with plugins, even though it is empty.